schemas/logs/salesforce/login-as.yml

```diff
@@ -0,0 +1,110 @@
+schema: Salesforce.LoginAs
+version: 0
+referenceURL: https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_objects_eventlogfile_loginas.htm
+description: "Login As events contain details about what a Salesforce admin did while logged in as another user."
+parser:
+  csv:
+    delimiter: ","
+    hasHeader: true
+fields:
+  - name: EVENT_TYPE
+    required: true
+    type: string
+    validate:
+      allow: ["LoginAs"]
+  - name: TIMESTAMP
+    required: true
+    type: string
+  - name: REQUEST_ID
+    required: true
+    type: string
+    description: >-
+      The unique ID of a single transaction. A transaction can contain one or more events.
+      Each event in a given transaction has the same REQUEST_ID.
+      For example: 3nWgxWbDKWWDIk0FKfF5DV.
+    indicators:
+      - trace_id
+  - name: ORGANIZATION_ID
+    required: true
+    type: string
+    description: "The 15-character ID of the organization. For example: 00D000000000123."
+  - name: USER_ID
+    required: true
+    type: string
+    description: >-
+      The 15-character ID of the user who’s using Salesforce services through the UI or the API.
+      For example: 00530000009M943
+  - name: RUN_TIME
+    required: true
+    type: bigint
+    description: "The amount of time that the request took in milliseconds."
+  - name: CPU_TIME
+    required: true
+    type: bigint
+    description: >-
+      The CPU time in milliseconds used to complete the request.
+      This field indicates the amount of activity taking place in the app server layer.
+  - name: URI
+    required: true
+    type: string
+  - name: SESSION_KEY
+    required: true
+    type: string
+    description: >-
+      The user’s unique session ID. You can use this value to identify all
+      user events within a session. When a user logs out and logs in again, a new session is started.
+      For example: d7DEq/ANa7nNZZVD.
+  - name: LOGIN_KEY
+    required: true
+    type: string
+    description: >-
+      The string that ties together all events in a given user’s login session.
+      It starts with a login event and ends with either a logout event or the user session expiring.
+      For example: GeJCsym5eyvtEK2I.
+  - name: DELEGATED_USER_NAME
```

Comment:
Should add the username indicator: https://github.com/panther-labs/panther-analysis/blob/master/schemas/logs/one_login/events.yml#L55-L56

Reply:
Keep forgetting those, sorry :|

Reply:
Done!